client/packages/requisitions/src/RnRForms/ListView/RnRFormCreateModal.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import { AppRoute } from '@openmsupply-client/config';
 import { SupplierSearchInput } from '@openmsupply-client/system';
 import { ProgramSearchInput } from './ProgramSearchInput';
-import { useCreateRnRForm, useSchedulesAndPeriods } from '../../api';
+import { useCreateRnRForm, useSchedulesAndPeriods } from '../api';
```

Comment:
Just need to update the post-create navigate call! 

![Screenshot 2024-08-07 at 5 19 56 PM](https://github.com/user-attachments/assets/eed3365d-0894-4a4f-9452-b30daeb895fb)

Reply:
yep. had done - just in the second commit.. so a bit too slow!